web_widget_table/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "web_widget_table",
```

Comment:
@eilst Name Should be `Web Datatable Widget`

Reply:
ok thannks